centaur/src/main/resources/standardTestCases/wdl_draft3/taskless_engine_functions/taskless_engine_functions.wdl

```diff
@@ -6,7 +6,9 @@ workflow taskless_engine_functions {
 
   Array[String] strings = ["a", "b"]
 
-  String filepath = "gs://not/a/real/file.txt"
```

Comment:
Are we losing test coverage by changing this to a local file path?

Reply:
This is a consequence of validating that the string is a valid path before `basenaming` it. This test was actively because the "local" config doesn't think that `gs://...` is a valid path.